integrations/block-data-api.php

```diff
@@ -30,11 +38,6 @@ class BlockDataApiIntegration extends Integration {
 	public function load(): void {
 		// Wait until plugins_loaded to give precedence to the plugin in the customer repo.
 		add_action( 'plugins_loaded', function() {
-			// Do not load plugin if already loaded by customer code.
-			if ( defined( 'VIP_BLOCK_DATA_API_LOADED' ) ) {
-				return;
-			}
-
 			// Load the version of the plugin that should be set to the latest version, otherwise if it's not found deactivate the integration.
```

Comment:
Even though it feels duplicative, the concern would be if we (or other code) unintentionally called `load()` multiple times, we could end up with problems.

```suggestion
			if ( $this->is_loaded() ) {
				return;
			}

			// Load the version of the plugin that should be set to the latest version, otherwise if it's not found deactivate the integration.
```

Reply:
Okay should we do same for `configure` because that can also be called directly?

Reply:
I think we should avoid duplication so removing this `is_loaded` from integrations code and adding it in each function where needed.

Reply:
Ignore the above comment ... it makes sense to keep `is_loaded()` inside `activate` function of base integration class.

Reply:
Added in https://github.com/Automattic/vip-go-mu-plugins/pull/4839/commits/4bc3fdeda3547e91dcbc6c8656666e72049b7e2f